osu.Game/Graphics/Containers/StoryboardContainer.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Game.Storyboards;
+using osu.Game.Storyboards.Drawables;
+
+namespace osu.Game.Graphics.Containers
+{
+    /// <summary>
+    /// A container that handles <see cref="Storyboard"/> loading, as well as applies user-specified visual settings to it.
+    /// </summary>
+    public class StoryboardContainer : UserDimContainer
+    {
+        private readonly Storyboard storyboard;
+        private DrawableStoryboard drawableStoryboard;
+
+        public StoryboardContainer(Storyboard storyboard)
+        {
+            this.storyboard = storyboard;
+            EnableUserDim.Default = true;
```

Comment:
i don't understand why this is here and why it is not needed in the base class. there's no explanation anywhere.

Reply:
I've added a comment, though I don't know if it would be better to make `EnableUserDim` exclusive to `DimmableBackgroundContainer` instead, since I'm not sure if this would change in the future.

Reply:
Your comment still explains nothing. Are you meaning to set the default in `UserDimContainer`? Because that seems like what you want to do. So i've done this.

Also is there actually a reason to change `Default`? It looks like it would never be used.

Reply:
should be fine to set the default in `UserDimContainer`. Thought it would be more sane if the default was set too, but its not being used right now.

Reply:
As discussed this is not relevant.